src/ctree-app/ctree-app.html

```diff
@@ -373,8 +394,8 @@
         dialog.open();
       }
 
-      _rootChanged(path) {
-        this._updateNavList();
+      clearSearch(){
```

Comment:
This should probably be marked as private by starting with `_` because I don't think we'll need to procedurally clear the search.